src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -37,3 +37,4 @@
 [assembly: InternalsVisibleTo("CommandLineTests")]
 [assembly: InternalsVisibleTo("Watch3DNodeModelsWpf")]
 [assembly: InternalsVisibleTo("DynamoPerformanceTests")]
+[assembly: InternalsVisibleTo("SampleViewExtension")]
```

Comment:
are you intending to remove this before merging?

Reply:
Yes, I will have to remove this. But how do we make sure, that this internal API is accessible from the view extension solution that the user's create?

Reply:
I think we can add a TODO to remove this when the API is actually public soon

Reply:
But this will put a constraint on the user that they need to name their assembly as "SampleViewExtension" so as to access this internal API. I am wondering if there is any better solution

Reply:
Well, to @mjkkirschner 's point, it is not meant for 3rd party developers to use right now, let's consume it internally so I will most likely update this to PackageDependencyViewExtension in my PR. And we need to finalize the API before we make it public, otherwise we could not break it once it is made public. Not easy!